zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/handlers/DecisionEvaluationHandlerTest.java

```diff
@@ -283,8 +286,14 @@ void shouldUpdateWithMultipleDecisionsOnCorrectState() {
             .from(factory.generateObject(DecisionEvaluationRecordValue.class))
             .withEvaluatedDecisions(
                 List.of(
-                    factory.generateObject(EvaluatedDecisionValue.class),
-                    factory.generateObject(EvaluatedDecisionValue.class)))
+                    ImmutableEvaluatedDecisionValue.builder()
+                        .from(factory.generateObject(EvaluatedDecisionValue.class))
+                        .withDecisionEvaluationInstanceKey(recordKey + "-1")
```

Comment:
🔧 why not using the generator here as well? when we decide at one point in the future to change the format, this test would fail.